README.md

```diff
@@ -19,3 +25,17 @@ Run `npm run test` to execute the unit tests via [Karma](https://karma-runner.gi
 ## Serve locally the generic web component
 
 If you need to see the render of the web component locally you can run `npm run preview:serve`.
+
+## How to use it in any frontend
+
+Install the library and declare component in template:
+
+<stc-app-widget></stc-app-widget>
+
+## Add github badges in the head of the file (github action status)
+
+Define actions in .github/workflows
```

Comment:
Non je parle d'ajouter un badge dans le readme du projet pour indiquer dans quel etat est le dernier build de la lib.
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Reply:
ok je mets le badge last build en haut de page